.github/workflows/bump.yaml

```
@@ -0,0 +1,18 @@
+name: Automatic version updates
```

I'd like to keep it consistent with existing workflows

```suggestion
name: Bump CI
```